README.md

```diff
@@ -101,10 +101,12 @@ This section has moved here: https://facebook.github.io/create-react-app/docs/ad
 
 ### Deploy the client:
 
-Deploy the entire repo to the server:
+Deploy the build to the server:
 
-```
-ansible-playbook -i staging -u buildout deploy.yml
-```
+* Run `npm run build` to create a build in the build folder
+
+* Go to the deploy folder
+
+* Run `ansible-playbook -i staging deploy.yml` to deploy the build to staging.
 
 NB: Assumed is that your ssh public key has been added to buildout's authorized_keys.
```

Comment:
_Assumed is that your ssh public key has been added to buildout's authorized_keys._

Dat klopt hier dus niet: we ssh-en als onszelf en sudo-en naar buildout.